frontend/src/pages/StatusPage/useDocuments.ts

```diff
@@ -0,0 +1,66 @@
+import { useCallback, useEffect, useState } from "react";
+import { SortingRule } from "react-table";
+import useAuthContext from "context/AuthContext";
+import { Document, ApiDocument } from "solo-types";
+import { createFakeApiDocs } from "solo-types";
+
+// covert api returned document to Document
+export const parseApiDocuments = (apiDocs: ApiDocument[]): Document[] =>
```

Comment:
Thoughts? Should there be tests for this and the `useDocuments`? 🤷‍♂

Reply:
The behavior is tested tested completely by testing the StatusPage component:

<img width="678" alt="image" src="https://user-images.githubusercontent.com/59582489/76336922-121e1300-62cd-11ea-83ad-685f5a754558.png">

I can add explicit tests too though

Reply:
Nah thats fine. I expected that it was covered. Nbd its all pretty normal stuff.